invenio_stats/aggregations.py

```diff
@@ -384,27 +422,35 @@ def run(self, start_date=None, end_date=None, update_bookmark=True):
         # If no events have been indexed there is nothing to aggregate
         if not dsl.Index(self.event_index, using=self.client).exists():
             return
-
+        if update_bookmark:
```

Comment:
This condition is not clear to me, why getting the current bookmark only if to be updated?

Reply:
I see, but then we need to change tests :) We should not change code because of tests.

Reply:
Ok, changed the tests. Now, they verify that the index exist, and that it has no documents